bakerytest/bakerytest_test.go

```diff
@@ -142,7 +145,11 @@ func (s *suite) TestInteractiveDischarger(c *gc.C) {
 	var d *bakerytest.InteractiveDischarger
 	d = bakerytest.NewInteractiveDischarger(nil, http.HandlerFunc(
 		func(w http.ResponseWriter, r *http.Request) {
-			d.FinishInteraction(w, r, nil)
+			d.FinishInteraction(w, r, []checkers.Caveat{
+				checkers.Caveat{
+					Condition: "test pass",
```

Comment:
Even better, make the checker save the condition tested so that you know that it's actually tested the condition you're expecting.